core-rust/state-manager/src/transaction/executable_logic.rs

```diff
@@ -64,43 +63,65 @@ impl ExecutionConfigurator {
     ) -> Self {
         Self {
             scrypto_vm: ScryptoVm::<DefaultWasmEngine>::default(),
-            costing_parameters,
             execution_configs: HashMap::from([
                 (
                     ConfigType::Genesis,
                     ExecutionConfig::for_genesis_transaction(network.clone())
+                        .with_costing_parameters(costing_parameters)
```

Comment:
Do we ever need to override costing parameters? Can we just get rid of this? (And instead use the new `disable_costing` if we need to?)